File: Sprint 03/Atividade 11/1.4/Tarefa.java
import java.util.Scanner;

public class Tarefa {
	public static void main(String[] args) {
		long number;
		Scanner in = new Scanner(System.in);

		System.out.printf("Digite um numero: ");
		number = in.nextLong();
	
		System.out.printf("Numero digitado: " + number + ".\n");
		System.out.printf("Fatorial deste numero: " + fatorial(number) + ".\n");
		in.close();
	}

	public static long fatorial(final long number) {
		long fatorial = 1;
		for(long i=2; i<=number; i++)
			fatorial *= i;

		return fatorial;
	}
}

File: Sprint 03/Atividade 11/1.4/tarefa.py
def fatorial(number):
	if (number <= 0):
		return 0

	fatorial = 1
	for i in range(2, (number+1)):
		fatorial = fatorial * i

	return fatorial

number = int(input("Digite um numero: "))
print("Numero digitado: {}.".format(number))

if(fatorial(number) == 0):
	print("Este numero nao tem fatorial.")
else:
	print("Fatorial deste numero: {}.".format(fatorial(number)))
